build/Dockerfile

```diff
@@ -1,12 +1,31 @@
-# Image building stage
-FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
+# Multi-stage docker build
+# Build stage
+FROM golang:1.14 AS builder
```

Comment:
We have planned to use golang version 1.16 for all the builds. This will be an update in:
- workflows
- here
- go.mod